strongbox-web-core/pom.xml

```diff
@@ -500,6 +500,11 @@
             <artifactId>springfox-swagger-ui</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-xmlrpc</artifactId>
+            <version>2.23.4</version>
```

Comment:
Versions should be defined in the `strongbox-parent` and inherited from there instead.